sources/camunda-docker-compose-core.yml

```diff
@@ -0,0 +1,99 @@
+# While the Docker images themselves are supported for production usage,
+# this docker-compose.yaml is designed to be used by developers to run
+# an environment locally. It is not designed to be used in production.
+# We recommend to use Kubernetes in production with our Helm Charts:
+# https://docs.camunda.io/docs/self-managed/platform-deployment/kubernetes-helm/
+# For local development, we recommend using KIND instead of `docker-compose`:
+# https://docs.camunda.io/docs/self-managed/platform-deployment/kubernetes-helm/#installing-the-camunda-helm-chart-locally-using-kind
+
+# This is a lightweight configuration with Zeebe, Operate, Tasklist, and Elasticsearch
+# See docker-compose.yml for a configuration that also includes Optimize, Identity, and Keycloak.
+
+services:
+
+  zeebe: # https://docs.camunda.io/docs/self-managed/platform-deployment/docker/#zeebe
+    image: camunda/zeebe:${CAMUNDA_PLATFORM_VERSION:-8.0.6}
+    container_name: zeebe
+    ports:
+      - "26500:26500"
+      - "9600:9600"
```

Comment:
What is the process owning port 9600?

Reply:
9600: Metrics and Readiness Probe. Prometheus metrics are exported on the route /metrics. There is a readiness probe on /ready.
Source: https://docs.camunda.io/docs/self-managed/zeebe-deployment/operations/network-ports/#broker